content/en/news/releases/1.11.x/announcing-1.11.2/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Announcing Istio 1.11.2
+linktitle: 1.11.2
+subtitle: Patch Release
+description: Istio 1.11.2 patch release.
+publishdate: 2021-09-02
+release: 1.11.2
+aliases:
+    - /news/announcing-1.11.2
+---
+
+This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.11.1 and Istio 1.11.2.
+
+{{< relnote >}}
+
+## Changes
```

Comment:
```
**Added** support for gRPC configuring workloads via xDS without an Envoy proxy. 
```

If I manage to get https://github.com/istio/istio.io/pull/10168 in then it would be good to link it.

Reply:
@stevenctl Okay, please ping me directly in slack if it can be in, then I will add it.

Reply:
Will not be merged in time